core/src/main/scala/org/apache/spark/sql/tispark/TiHandleRDD.scala

```diff
@@ -50,6 +52,13 @@ class TiHandleRDD(val dagRequest: TiDAGRequest,
 
   override def compute(split: Partition, context: TaskContext): Iterator[Row] =
     new Iterator[Row] {
+      if (!tiConf.checkLocalTimeZone()) {
+        throw new TiInternalException(
+          "timezone are different! dirver: " + tiConf.getLocalTimeZone + " executor:" + Converter.getLocalTimezone +
+          " please set spark.driver.extraJavaOptions=-Duser.timezone=GMT+8 and spark.executor.extraJavaOptions -Duser.timezone=GMT+8"
```

Comment:
I don't think +8 is the only choice for all users

Reply:
+1

Reply:
error message is an example

Reply:
change to `spark.driver.extraJavaOptions=-Duser.timezone=${some time zone}` ?

Reply:
I think suggesting "please set user.timezone in spark.driver.extraJavaOptions and spark.executor.extraJavaOptions" is enough

Reply:
let the users decide what values to be set.